<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Drive Data Dynamics - Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	border-radius: 15px;
}

QWidget:enabled {
	background-color: rgb(0, 85, 0);
}

QWidget:disabled {
	background-color: rgba(0, 50, 0, 200);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+Z</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>710</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget {
	border-radius: 15px;
}

QWidget:enabled {
	background-color: rgb(170, 0, 0);
}

QWidget:disabled {
	background-color: rgba(140, 0, 0, 200);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+X</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kickButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>710</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	border-radius: 15px;
}

QWidget:enabled {
	background-color: rgb(0, 0, 0);
}

QWidget:disabled {
	background-color: rgba(0, 0, 0, 200);
}</string>
    </property>
    <property name="text">
     <string>   Kick All</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+K</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="statusBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>580</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="extra3StatusLight">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>165</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/off.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="extra2StatusLight">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>130</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/off.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="connectionStatusLight">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/off.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="databaseStatusLight">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/off.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="extra1StatusLight">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>95</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/off.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="connectionText">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>25</y>
       <width>70</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
    <widget class="QLabel" name="databaseText">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>70</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QLabel" name="extra1Text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>95</y>
       <width>70</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Extra 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="extra2Text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>70</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Extra 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="extra3Text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>165</y>
       <width>70</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Extra 3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connectionBox">
    <property name="geometry">
     <rect>
      <x>869</x>
      <y>19</y>
      <width>301</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Connections</string>
    </property>
    <widget class="QListWidget" name="connectionsList">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>291</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget::item { 
	text-align: center; 
	width: 100%;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::DropAction::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ViewMode::ListMode</enum>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignmentFlag::AlignRight</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="databaseFrame">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>19</y>
      <width>751</width>
      <height>521</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 255);

border: 10px solid;

border-top-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0.106145 rgba(85, 85, 255, 255), stop:1 rgba(0, 0, 0, 255));
border-bottom-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.893855 rgba(85, 85, 255, 255));
border-right-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.893855 rgba(85, 85, 255, 255));
border-left-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0.106145 rgba(85, 85, 255, 255), stop:1 rgba(0, 0, 0, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
   </widget>
   <widget class="QTabWidget" name="settingsTabs">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>580</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="connectionTabs">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
     <widget class="QLabel" name="portInputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="portInputBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
      </property>
     </widget>
     <widget class="QLabel" name="timeoutInputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Timeout:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="timeoutInputBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
      </property>
     </widget>
     <widget class="QLabel" name="hostInputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostInputBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="connectionSaveButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 0);</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Other</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>587</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="carImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>231</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/Images/carDesign1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carImage2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>173</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/Images/carDesign2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>kickButton</tabstop>
  <tabstop>connectionsList</tabstop>
  <tabstop>settingsTabs</tabstop>
  <tabstop>hostInputBox</tabstop>
  <tabstop>portInputBox</tabstop>
  <tabstop>timeoutInputBox</tabstop>
  <tabstop>connectionSaveButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
